auto3dseg/algorithm_templates/swinunetr/scripts/train.py

```diff
@@ -183,15 +182,15 @@ def run(config_file: Optional[Union[str, Sequence[str]]] = None, **override):
     CONFIG["handlers"]["file"]["filename"] = log_output_file
     logging.config.dictConfig(CONFIG)
 
-    logger.debug(f"[info] number of GPUs: {torch.cuda.device_count()}")
+    logger.debug(f"[debug] number of GPUs: {torch.cuda.device_count()}")
```

Comment:
the formatter defined here should have already included the levelname `debug` so `[debug]` is unecessary
https://github.com/Project-MONAI/research-contributions/blob/5c91e57cf3d03adc3e12766a0a49e8847aa10f94/auto3dseg/algorithm_templates/swinunetr/scripts/train.py#L79

could you please double-check the log? cc @mingxin-zheng

Reply:
I assume the `[debug]` can be deleted.

INFO has the following format: 
```
2023-05-18 06:16:18,179 - INFO - Generated:/tmp/tmp5mxyr3p8/workdir/swinunetr_0
```

Reply:
Deleted